extensions/machine-learning/src/common/constants.ts

```diff
@@ -175,7 +175,12 @@ export const azureModels = localize('models.azureModels', "Models");
 export const azureModelsTitle = localize('models.azureModelsTitle', "Azure models");
 export const localModelsTitle = localize('models.localModelsTitle', "Local models");
 export const modelSourcesTitle = localize('models.modelSourcesTitle', "Source location");
-export const modelSourcePageTitle = localize('models.modelSourcePageTitle', "Where is your model located?");
+export const modelSourcePageTitle = localize('models.modelSourcePageTitle', "Select model source type");
+export const localModelSourceDescriptionForImport = localize('models.localModelSourceDescriptionForImport', "‘File Upload’ is selected. This allows you to import a model file from your local machine into a model database in this SQL instance. Click ‘Next’ to continue.​");
```

Comment:
Might be a far fetched ask. Can we not make "‘File Upload’ is selected - a common string with the label being the argument?

Reply:
+ @MsSQLGirl for feedback.